#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_add_bash_to_centcore.dpatch by  <ctemple@srvi-packaging-ubuntu-i386.merethis.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add bash to centcore init script

@DPATCH@

--- SOURCE_DIR/tmpl/install/centcore.init.d	2010-03-02 18:06:09.000000000 +0100
+++ SOURCE_DIR/tmpl/install/centcore.init.d.patch	2010-03-02 18:07:23.000000000 +0100
@@ -138,7 +138,7 @@
 	    # Test if running directory exist.
 	    rundir_exist
 	    echo "Starting Centcore"
-	    su - @NAGIOS_USER@ -c "nice -n $NICE $Bin >> $DemLog 2>&1"
+	    su - @NAGIOS_USER@ -s /bin/bash -c "nice -n $NICE $Bin >> $DemLog 2>&1"
 	    if [ -d $LockDir ]; then 
 	    	touch $LockDir/$LockFile; 
 	    fi
